<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BrajR" name="com.edureka.scripts.VerifyCourseAmountOnOrderSummaryForForeignCurrency" tests="1" failures="1" timestamp="13 Jun 2016 17:08:08 GMT" time="149.440" errors="0">
  <testcase name="test_07VerifyCourseAmountOnOrderSummaryForForeignCurrency" time="149.440" classname="com.edureka.scripts.VerifyCourseAmountOnOrderSummaryForForeignCurrency">
    <failure type="java.lang.AssertionError" message="Element Locator :(//div[contains(text(),&#039;Price :&#039;)])[2]/following::div/span[@id=&#039;org_price&#039;] Not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element Locator :(//div[contains(text(),'Price :')])[2]/following::div/span[@id='org_price'] Not found expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.edureka.util.DriverHelper.getText(DriverHelper.java:386)
	at com.edureka.pages.OrderSummaryPage.verifyTotalAmount(OrderSummaryPage.java:67)
	at com.edureka.scripts.VerifyCourseAmountOnOrderSummaryForForeignCurrency.test_07VerifyCourseAmountOnOrderSummaryForForeignCurrency(VerifyCourseAmountOnOrderSummaryForForeignCurrency.java:73)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
    </failure>
  </testcase> <!-- test_07VerifyCourseAmountOnOrderSummaryForForeignCurrency -->
</testsuite> <!-- com.edureka.scripts.VerifyCourseAmountOnOrderSummaryForForeignCurrency -->
